src/util/jira-utils.ts

```diff
@@ -105,7 +105,7 @@ export const jiraIssueKeyParser = (str: string): string[] => {
 	let regex = /(^|[^\p{L}\p{Nd}])([\p{L}][\p{L}\p{Nd}_]{1,255}-\p{Nd}{1,255})/giu;
 
 	if (issueKeyRegexCharLimitFeature) {
-		regex = /(^|[^A-Z\d])([A-Z][A-Z\d]{1,255}-[1-9]\d{1,255})/giu;
+		regex = /(^|[^A-Z\d])([A-Z][A-Z\d]{1,255}-[1-9]\d{0,255})/giu;
```

Comment:
some tests would be good, as I don't understand why it was broken and how that `0` fix it.

Reply:
yeah, I had added tests, but because of the FF and since this is a non-async function, there's no way of consistently added the FF to the test, so I went without it.

The 0 here depics how many characters are needed for the id.  The `[1-9]` is any number except 0, then `\d` is any number as long as it has the length of 0 to 255, where before it was excpecting at least another character so the id has to be 2 numbers for it to be caught by parser.